File: Timestamp_Party.c
#include <stdio.h>
#include <time.h>
#include <unistd.h>

#define     AWAITED_ONE   1500000000
#define     RING          printf("%c",7) ;
#define     TIMESTAMP     (unsigned)time(NULL)

int main (int argc, char** argv)
{
    int ctsp ;

    for (;;) {
        ctsp = TIMESTAMP ;
        if (ctsp == AWAITED_ONE)
        {
            break ;
        }
        sleep(0.3) ;
    }

    RING ;
    printf(
        "\n%s\n",
        "************************\n"
        "     Merry Timestamp !  \n"
        "        1500000000      \n"
        "************************"
    ) ;
    RING ;
}

File: Timestamp_Party.py
# -*- coding: utf-8 -*-
# author: pBouillon - https://github.com/pBouillon

import time
from time import sleep
from time import time

def beep():
    print ('\a')

if __name__ == '__main__':
    awaited_timestamp = 1500000000
    ctsp = 0
    waiting = True

    while (waiting) :
        ctsp = int(time())
        if ctsp == awaited_timestamp :
            waiting = False
        sleep(.3)

    beep ()
    print ("""\n
    ************************\n
         Merry Timestamp !  \n
            1500000000      \n
    ************************"
    \n""")
    beep ()
